includes/admin.inc

```diff
@@ -618,8 +618,8 @@ function islandora_solr_admin_settings($form, &$form_state) {
     '#type' => 'textarea',
     '#title' => t('Query fields'),
     '#default_value' => variable_get('islandora_solr_query_fields', 'dc.title^5 dc.subject^2 dc.description^2 dc.creator^2 dc.contributor^1 dc.type'),
-    '#description' => t('<a href="@url" target="_blank" title="Solr query fields documentation">Query fields</a> to use for DisMax (simple) searches.', array(
-      '@url' => 'http://wiki.apache.org/solr/DisMaxQParserPlugin#qf_.28Query_Fields.29',
+    '#description' => t('Query fields to use for DisMax (simple) searches. See the QueryFields section in the <a href="@url" target="_blank" title="Solr documentation">Solr documentation</a> for examples.', array(
```

Comment:
Since you are here. "DisMax or eDisMax"?

Reply:
Its a question really, dismax and edismax, both are valid options. And if you are using edismax, those apply too. But not sure if that should go there?

Reply:
DisMax is what was originally there... knowing the difference is beyond my pay grade.

Reply:
man, you get actually paid! 😆  Ok, but people managing a repository and using that could know the difference? https://cwiki.apache.org/confluence/display/solr/The+Extended+DisMax+Query+Parser . Not a blocker anyway